tox.ini

```diff
@@ -78,6 +78,14 @@ changedir = {toxinidir}/tasks
 extras =
 commands = python update_embedded.py
 
+[testenv:zipapp]
+basepython = python3
+description = make a redistributable virtualenv zipapp
+skip_install = true
+changedir = {toxinidir}/tasks
+extras =
+commands = python make_zipapp.py --root ..
```

Comment:
we probably generate this into toxs tmp folder to not pollute the working tree

Reply:
I assume the person that would be manually running this would actually want to find the file so they could upload it somewhere -- I don't expect this to be run very often

Reply:
This gets run at every commit (it's part of the default target). It's same as documentation if you want to find it just print the path after generating it (or part of the generation).

Reply:
yeah you told me to add it to the default :P

I don't think it should be in the default, it's similar to `release` imo